src/platform/src/Contract/JsonPathConverter/ResultExtractorInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
To be removed ;)

Reply:
you're faster reviewing than myself - need to change my file template for this project

Reply:
Let's add rector and if not available, create an extra rule.

cc @votrubat